### ISSUE TITLE
Option -p discards some characters

### ISSUE THREAD

**Issue by aminophen** [aminophen/dviasm#13]
With option `-p` (ISO-2022-JP-encoded DVI for Japanese pTeX), some characters are removed.  It seems that bullet (0x88) is discarded by DecodeISO2022JP().

### Original DVI

Same as aminophen/dviasm#12.

### After processing with DVIasm

```
$ dviasm -p test.dvi >test-out.txt
$ dviasm -p -o test-out.dvi test-out.txt
$ dvipdfmx test-out.dvi
```

<img width="304" alt="20200830-dviasm-2" src="https://user-images.githubusercontent.com/9957466/91656899-80fb5c80-eaf7-11ea-8615-a632416e16c2.png">

**Comment by aminophen** [comment]
First attempt to fix this pTeX issue is available at f8af231 (branch 'ptex-gh13').

It seems that the original script by Jin-Hwan Cho had no support for this use case, so I'm not sure whether this is a correct implementation.

**Comment by aminophen** [comment]
Anyway I will release the new version including this change (f8af231).
